tensorflow/compiler/xla/service/platform_util.cc

```diff
@@ -203,6 +211,18 @@ static bool IsDeviceSupported(se::StreamExecutor* executor) {
         return false;
       }
     }
+  } else if (executor->platform()->id() == se::rocm::kROCmPlatformId) {
```

Comment:
Is this change tightly coupled to the previous one? It's much more straightforward, if it's not coupled, I would have it in a separate PR.